<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>421</width>
      <height>118</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Kinect Stream Server</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Camera 1 location:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Camera 2 location:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Camera 3 location:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLineEdit" name="camera1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="camera2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="camera3"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="c1btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="c2btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="c3btn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QPlainTextEdit" name="log">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>951</width>
      <height>791</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="GLwidget" name="camera1gl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="GLwidget" name="camera2gl">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="GLwidget" name="camera3gl">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="GLwidget" name="camera4gl">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>450</y>
       <width>281</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>401</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="fusion_btn">
       <property name="text">
        <string>fusion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pausebtn">
       <property name="text">
        <string>pause</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLwidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fusion_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fusion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLoadBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pausebtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClickedPauseBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>602</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fusion()</slot>
  <slot>onLoadBtn()</slot>
  <slot>onClickedPauseBtn()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
